### DISCUSSION TITLE
[Keyball44]左右の接触不良、右のReset反応しない

### DISCUSSION THREAD

**Discussion by don-wang** [Yowkees/keyball#632] [Q&A] [answered]
お世話になっております。
この前までは問題なく動作していますが、最近から使っている途中で、片方が反応しなくなります。
特に右側でPCと接続する時に、左側の接続問題が頻繫に発生しています。

また、右側のResetボタンがいつの間にか反応しなくなりました。

背面の写真をご参考ください。
アドレスをお願いします。

右側：
![keyball-b-r](https://github.com/user-attachments/assets/bc5aeace-da10-40c6-a5fb-3030b5442dcb)

左側：
![keyball-b-l](https://github.com/user-attachments/assets/5b03a3b9-1f98-4aca-9285-123409b0e7be)

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。

お写真を見ると、リセットスイッチとTRRSコネクタの半田付けで、はんだが浮いているように見える端子があります。
ボトムプレートを取り外して、半田付けし直してみてください。
はんだが基板の銀パッドになじんで、山型になるのが理想です。

お手数ですが、ご対応お願いいたします。

**Reply by don-wang** [reply]
なかなか時間取れなくて、ようやく半田付け直しできました。
現在は問題なく、快適に使っています。
ちなみに、リセットスイッチだけで治せず、マイコンRSTピンもやってみたら改善しました。

**Comment by don-wang** [comment]
迅速なご回答ありがとうございます。
問題を承知しました。半田付け直してみます。
